utils/flags/flags.go

```diff
@@ -51,6 +51,12 @@ func GetCliFlags() []cli.Flag {
 			Usage:   "The maximum size of bazel-remote's disk cache in GiB. This flag is required.",
 			EnvVars: []string{"BAZEL_REMOTE_MAX_SIZE"},
 		},
+		&cli.Int64Flag{
+			Name:    "disk_size_limit",
```

Comment:
I think we should find a better name for this, disk_size_limit is too similar to max_size. What do you think about max_size_hard_limit ?

Reply:
I have been thinking about renaming the original `max_size` to something with the word “target” or “eviction” since bazel-remote is evicting to reach that target, but is not 100% enforcing it as a hard max limit. Examples of pairs:

`target_disk_size`   `max_disk_size`

`disk_size_target`   `disk_size_limit`

`eviction_disk_size`   `rejection_disk_size`

The word “disk” could help emphasize that it is not max request size, not max amount stored in proxy backend, etc. 

However, I assume it is not motivated to do such non-backward-compatible renaming and I think your suggested name `max_size_hard_limit` aligns well with the existing `max_size`.

I’m working on adding unit tests, and **I will update according to your suggestions** when I’m ready with the unit tests.